VAMobile/documentation/docs/Flagship design library/Templates/Home.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Home Screen
+sidebar_position: 4
+---
+
+The Home screen is the initial/default screen of the VA Health & Benefits mobile app. It sits at the top level of the hierarchical navigation (as a peer to the Health, Benefits and Payments category screens) and displays a personalized summary of a Veteran's current tasks and interactions with features in the VA mobile app. It also contains the primary entry point for the Veteran's Profile information (including App Settings).
+
+<iframe width="500" height="800" alt="image of VA mobile app Home screen" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fdesign%2Fcdp7Be4UdYesq9fXeqaOgt%2FNavigation-2.0-Screen-Templates---%F0%9F%9A%A2-Shipped---VA-Mobile%3Fm%3Dauto%26node-id%3D7733-11182%26t%3DGsZrpZDn1qDXDTr0-1" allowfullscreen></iframe>
+
+## Anatomy:
+
+The VA mobile app’s personalized home screen combines content that is variable, fixed, personalized, and evergreen to deliver an experience that surfaces value to Veterans and supports understanding of the app’s capabilities, regardless of what a Veteran’s interactions are with the VA. To do this in a way that's predictable and performant, the Home screen displays a mix of content types within predetermined locations (zones).
+
+Having a mix of content, arranged into intentional zones, ensures that users:
+
+
+
+* Always see elements that help demonstrate the current & future value of the app
+* Always see "personalized" information even if they don't have any active tasks at a given moment (users don’t experience an empty home screen in the absence of tasks)
+* See the things that are likely relevant and valuable to them first
+* Get a consistent information structure for screen readers to tab through that is present even if load times lag
+* See things that are important to proper app functioning when necessary
+
+
+### Zones
+
+The VA Mobile app Home screen zones ensure that the most impactful content items are getting the proper placement in the screen's hierarchy while also handling a variety of user contexts and technical situations. 
+
+<img src="/workspaces/va-mobile-app/VAMobile/documentation/static/img/home/vamobile-home-zones.png" width="700" alt="Diagram of home screen zones for VA mobile app">
```

Comment:
```suggestion
<img src="/workspaces/va-mobile-app/VAMobile/documentation/static/img/home/vamobile-home-zones.png" width="700" alt="Diagram of home screen zones for VA mobile app" />
```

Reply:
I need to able to control the width & height, so I used an img tag

Reply:
Can the image be resized instead?